crates/sui-verifier-transactional-tests/tests/one_time_witness/wrong_field_type_with_init.move

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// invalid, wrong struct field type
+
+//# init --addresses test=0x0
+
+//# publish
+module test::m {
+
+    struct M has drop { value: u64 }
```

Comment:
Maybe just duplicate this test to make sure we can still construct this module

Something like 
```
test::m {
struct M has drop { value: u64 }

fun foo() {
  M { value: 0 };
  M { value: 1 };
}
}

Reply:
I kind of already did (in `wrong_field_type.move` above but admittedly without explicit instantiation which I now added.